NightscoutServiceKit/RemoteCommands/V1/Notifications/BolusRemoteNotification.swift

```diff
@@ -31,11 +31,8 @@ public struct BolusRemoteNotification: RemoteNotification, Codable {
         return .bolusEntry(BolusAction(amountInUnits: amount))
     }
     
```

Comment:
These changes to the validator were a minor refactor as I'm adding additional "validation" logic in this PR. 

I didn't end up using the refactor pieces for this bug fix but I think the refactor is still useful.